convert.go

```diff
@@ -13,6 +13,10 @@ var errIncorrectNetAddr = fmt.Errorf("incorrect network addr conversion")
 
 // FromNetAddr converts a net.Addr type to a Multiaddr.
 func FromNetAddr(a net.Addr) (ma.Multiaddr, error) {
+	if a == nil {
+		return nil, fmt.Errorf("nil address passed to 'FromNetAddr'")
```

Comment:
maybe just `"nil multiaddr"` ? people can search, small enough module, and slightly less esoteric err message